### ISSUE TITLE
Plugin Ordering Issue

### ISSUE THREAD

**Issue by matthewwithanm** [matthewwithanm/jquery.picplus#23]
Using both jquery.picplus.queueup and jquery.picplus.lazyload, I was getting a really confusing error:

```
Uncaught TypeError: Object #<Object> has no method 'promote'
```

It turns out it was because the lazyload plugin was being initialized first and triggering a load before the queueup plugin had a chance to override the load methods.

I'm not sure if this is a documentation issue or we should address this in the code. At first I was going to change the lazyload plugin to use a timeout but that seems undesirable. Maybe plugins should have a two-phase initialization, only the second phase of which should do anything that might trigger a load?

/cc @lettertwo

**Comment by lettertwo** [comment]
A two phase initialization seems like it might work. What would they be called?  "install" and "initialize" or something?

**Comment by matthewwithanm** [comment]
Ooh, that's perfect. Much better than what I was thinking. :thumbsup:

**Comment by matthewwithanm** [comment]
Closed by f5aea16b143423f4e3e0a0d24ce64aef2afdc21a.
